pkg/service/test/util.go

```diff
@@ -102,15 +103,56 @@ func AddHttpBodyToMap(body string, m map[string][]string) error {
 	return nil
 }
 
+func JoinNoises(globalNoise models.GlobalNoise, tsNoise models.GlobalNoise) models.GlobalNoise {
+	noise := globalNoise
+	for k1, v1 := range tsNoise {
```

Comment:
same here k1 and k2 and check if for loop is needed

Reply:
done!